authemail/views.py

```diff
@@ -25,7 +25,7 @@ class Signup(APIView):
     serializer_class = SignupSerializer
 
     def post(self, request, format=None):
-        serializer = self.serializer_class(data=request.data)
+        serializer = self.serializer_class(data=request.data, context={'request': request})
```

Comment:
Would it be easier / better to call `context = self.get_serializer_context()` here?

(honest question, I use viewsets 99% of the time so don’t remember if view classes have the method)

Reply:
Good spot, you're right!

In fact, `self.get_serializer(data=request.data)` should just be used instead. That defined in `generics.GenericAPIView` and calls through to `get_serializer_context()` so we don't have to.

Reply:
Edit: nice idea, but `get_serializer` and `get_serializer_context` are only present on `generics.GenericAPIView` so it won't work here.